Urania/Urania.Core/WdCalculator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace Urania.Core {
+    public class WdCalculator {
+        public static decimal Calculate(decimal? id, decimal? ar, decimal? od) {
+            if (id < 0) {
+                throw new ArgumentOutOfRangeException($"{nameof(id)} cannot be less than 0");
```

Comment:
Prawidłowa implementacja tego wyjątku powinna wyglądać tak:
`throw new ArgumentOutOfRangeException(nameof(id), id, "Cannot be less than 0");`